HeadModel - Final Part

Hicham2012 opened the pull request.

Hicham2012 committed:
MODIFY: HeadModel
- Test other 3d models.
- Fix some typos.

tkalejandro commented:
@Hicham2012  There is something wrong in this PR..  maybe try to run `npm run build` and check locally what is actually wrong.

tkalejandro commented on src/modules/Experience/models/HeadModel.tsx:
```diff
@@ -13,21 +13,40 @@ interface HeadModelProps extends GroupProps {
+  const [gltfScene1, setGltfScene1] = useState<Group<Object3DEventMap> | null>(null);
+
   const { scene } = useGLTF('/models/head-3.glb');
+  const { scene1 } = useGLTF('/models/cone.glb')
```
MAybe is this scene1 your bug... are you sure scene1 exist inside this GLTF??

If you need many scenes maybe, dont destructure , you can try something like.  coneModel.scene  or as you wish.

Hicham2012 replied:
I fixed the bug, thanks :)

Hicham2012 committed:
Save Changes
- Add eyes.
- Minimize model vertices.

Hicham2012 committed:
Save changes

Hicham2012 committed:
Merge branch 'main' into 129-headmodel---final-part

Hicham2012 committed:
Latest Changes
- Add hair.

Hicham2012 committed:
Merge branch 'main' into 129-headmodel---final-part

Hicham2012 committed:
Merge branch 'main' into 129-headmodel---final-part

Hicham2012 committed:
MODIFY: AboutScene
- Fix colors.

Hicham2012 committed:
MODIFY: PointsLoader and Message
- Rmoved bug when clicking on a button in about section.
- Commented Clouds temporary till we optimize them.

Hicham2012 committed:
SAVE changes of MovingFace

Hicham2012 committed:
MODIFY: AboutScene.tsx
- Changed headmodel default color.

Hicham2012 committed:
MODIFY: PointsLoader.tsx
- Fix model rotation.

Hicham2012 committed:
Clean code.

tkalejandro commented on src/modules/Experience/scenes/WelcomeScene/components/Message/Message.tsx:
```diff
@@ -93,7 +93,7 @@ const Message = () => {
         </Box>
       </Flex>
-      <Clouds material={THREE.MeshBasicMaterial}>
+      {/* <Clouds material={THREE.MeshBasicMaterial}>
```
So what is happening with this clouds??  

Are they in or out?? XD 


Over all did you test your solution mobile tablet and desktop?

Hicham2012 replied:
The clouds make the website break for me.
I made sure that solution is working on mobile and other devices in the next commit.

tkalejandro replied:
So there are no clouds anymore ???

Hicham2012 replied:
We will keep them, but will optimize them in a separate ticket.

Hicham2012 committed:
MODIFY: PointerLoader, AboutScene, HeadModel
- Make model not clickable on mobile.
- Fix theme colors.
